addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -567,6 +567,7 @@ export class LinkPlugin extends Plugin {
                 this.dependencies.selection.focusEditable();
             },
             onEdit: () => {
+                this.LinkPopoverState.editing = true;
```

Comment:
Do we really need it? The `onEdit` is bond to the editing button in the popover to control which UI the popover should have. It's to initialize the popover state value `editing`. The editing value is set in `onClickEdit` before calling `onEdit` so I think we probably don't need to set it here